reference/3.0/Microsoft.PowerShell.Management/Test-Path.md

```diff
@@ -308,6 +308,17 @@ The acceptable values for this parameter are:
 
 Tells whether the final element in the path is of a particular type.
 
+> [!CAUTION] 
+> 
+> Up to PowerShell version 6.1.2, when the **IsValid** and **PathType** switches are 
+> specified together, the `Test-Path` cmdlet simply  the **PathType** switch and only 

```

Comment:
Looks like a word got removed between "simply" and "the **PathType**".  Also consider using a different word than simply.  Carry forward on versions.

Reply:
fixed